src/test/suite/fixtures/recorded/languages/markdown/changeName.yml

```diff
@@ -0,0 +1,29 @@
+languageId: markdown
```

Comment:
Note how this one now works from within the body of the section rather than just from within the heading; also note how it doesn't include the space anymore